vmm/src/api/openapi/cloud-hypervisor.yaml

```diff
@@ -999,6 +1001,19 @@ components:
           type: boolean
           default: false
 
+    DebugConsoleConfig:
+      required:
+        - mode
+      type: object
+      properties:
+        file:
+          type: string
+        mode:
+          type: string
+          enum: [Off, Pty, Tty, File, Socket, Null]
```

Comment:
Why exposes `Socket` while not support it?

Reply:
Due to the uncertainty described earlier:

> PS: Documentation and code partially look like --serial and --console can use a socket. But I can't find where this is implemented in the code - looks like this was never done or removed. This should be fixed either in documentation or be implemented.

Will fix it

Reply:
I think it's reasonable to list fewer modes for the `DebugConsoleConfig` for in the OpenAPI metadata.

Reply:
Done and  rebased.